find/finder.go

```diff
@@ -916,6 +916,13 @@ func (f *Finder) DefaultFolder(ctx context.Context) (*object.Folder, error) {
 	}
 	folder := object.NewFolder(f.client, ref.Reference())
 
+	// Set the InventoryPath of the newly created folder object
+	e, err := f.Element(ctx, folder.Reference())
```

Comment:
In this case, we can avoid calling `f.Element`, since we have the Datacenter already and the vmFolder name is always "vm".  I think the following should be ok:
```go
folder.SetInventoryPath(path.Join(f.dc.Datacenter.InventoryPath, "vm"))
```
Can you try that out?

Reply:
I tried this approach and it works. Pushed a new commit with the change.